pkg/virt-controller/services/template_test.go

```diff
@@ -474,14 +474,18 @@ var _ = Describe("Template", func() {
 				pod, err := svc.RenderLaunchManifest(&vmi)
 				Expect(err).ToNot(HaveOccurred())
 
-				Expect(len(pod.Spec.InitContainers)).To(Equal(1))
+				Expect(len(pod.Spec.InitContainers)).To(Equal(2))
```

Comment:
I would feel a little bit more comfortable if we would also have a unit test with two container disks.

Reply:
:+1:  alrighty

Reply:
fixed